EpikCord/client/buckets.py

```diff
@@ -10,18 +10,26 @@
 
 
 class MockBucket:
+    """A mock bucket that does nothing."""
     async def wait(self):
+        """Pretends to wait for the Event to be set"""
         ...
 
     def clear(self):
+        """Pretends to clear the Event"""
```

Comment:
*"The docstring is a phrase ending in a period."*
According to the [pep 8](https://peps.python.org/pep-0008/), docstring should be *"consistent with the docstring convention in [pep 257](https://peps.python.org/pep-0257)"*.

Reply:
Took a look at PEP257 and I need to do some major changes to docstrings, majorly making them commands.

Reply:
I'm not sure on how much you should follow it, though, making docstring "a phrase ending in a period." would definitively improve them.